core/server/api_container/server/startosis_engine/kurtosis_starlark_framework/kurtosis_plan_instruction/kurtosis_plan_instruction.go

```diff
@@ -92,13 +110,20 @@ func (builtin *KurtosisPlanInstructionWrapper) CreateBuiltin() func(thread *star
 				logrus.Debugf("Marking the plan as invalid as instruction '%s' differs from '%s'",
 					instructionWrapper.String(), scheduledInstructionPulledFromMaskMaybe.GetInstruction().String())
 			}
+			kurtosisInstructionStr := scheduledInstruction.GetInstruction().String()
+			enclaveCapabilities := scheduledInstruction.GetInstruction().GetCapabilites().GetEnclavePlanCapabilities()
```

Comment:
This chain `.GetInstruction().GetCapabilites().GetEnclavePlanCapabilities()` is blowing my mind 🤯